runtime/executor/method.cpp

```diff
@@ -1512,6 +1513,26 @@ Error Method::experimental_step() {
   return step();
 }
 
+Error Method::update(executorch::runtime::ArrayRef<executorch::runtime::Entry> backend_option) {
+  for (const auto& entry : backend_option) {
+    const char* backend_name = entry.backend_name;
+    auto backend_options = entry.options;
+
+    auto backend_class = get_backend_class(backend_name);
```

Comment:
does backend class give you a singleton instance of backend interface? If so why do we need to pipe it up via Method class. Just make something like `executorch::backend::set_options(BackendOptions&)` (I dont think BackendOptions constructor exist? but probably would be good to add)

Reply:
I agree. Allowing it to pass via Method or Module hints that it is private to a method which in fact it isn't.